Makefile

```diff
@@ -11,7 +11,7 @@ help: ## Show this help
 	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[33m%-20s\033[0m %s\n", $$1, $$2}'
 
 run: ## Start up both client and server containers
-	docker-compose up --build
+	docker-compose up --build --force-recreate
```

Comment:
why do we need this?

Reply:
Docker would keep reusing cached layers of package.json even though I had just installed packages, it got a bit annoying